### PR TITLE
Testcase

### PR DESCRIPTION
**Pull request by EJEunjiyaz**

### CODE REVIEW

**Review by Klan300 — APPROVED** [review]

**Review by Parkkung — APPROVED** [review]

**Review by kornkritpawit — APPROVED** [review]
approve

**Review by Parkkung — CHANGES_REQUESTED** [review]
have a error in test, please fix it

**Review by Parkkung — APPROVED** [review]
everything is ok now

**Review by Klan300 — CHANGES_REQUESTED** [review]
please look on travis ci and find problem before 12/12/19

**Comment by EJEunjiyaz** [comment]
> please look on travis ci and find problem before 12/12/19

Thank you. The bug fixed on 10/12/19. Final check, please

File: Cardgame/tests.py [hunk 1]
```diff
@@ -1,11 +1,63 @@
...
+        self.deck2 = Deck.objects.create(deck_name='Science')
+        self.card1 = Card.objects.create(deck=self.deck1, card_name='What is the alphabet?')
     
     def test_deck_count(self):
         self.assertEqual(2, Deck.objects.count())
+
+    def test_card_count(self):
+        english = Card.objects.get(deck=self.deck1)
+        self.assertEqual(english.deck, self.deck1)
+
```
Comment (Klan300) [inline Cardgame/tests.py:25]: pls test method number of card of deck models

Comment (Klan300) [inline Cardgame/tests.py:25]: test __str__ method of card models too

**Review by Klan300 — CHANGES_REQUESTED** [review]

**Review by Klan300 — APPROVED** [review]

**Pull request merged**